contracts/ConstantPriceFeed.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "./vendor/@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+contract ConstantPriceFeed is AggregatorV3Interface {
+    /** Custom errors **/
+    error InvalidInt256();
+    error NotImplemented();
+
+    /// @notice Version of the price feed
+    uint public constant override version = 1;
+
+    /// @notice Description of the price feed
+    string public constant description = "Constant price feed";
+
+    /// @notice Number of decimals for returned prices
+    uint8 public immutable override decimals;
+
+    /// @notice The constant price
+    int private immutable CONSTANT_PRICE;
+
+    /**
+     * @notice Construct a new scaling price feed
+     * @param decimals_ The number of decimals for the returned prices
+     **/
+    constructor(uint8 decimals_) {
+        decimals = decimals_;
+        CONSTANT_PRICE = signed256(10 ** decimals_);
```

Comment:
Kinda wonder if we should just pass this in, or a factor... or otherwise maybe rename `UnitPriceFeed`, but arbitrary constant seems more flexible

Reply:
Yeah good point, passing in an arbitrary constant sounds good. I'll make that change.

Reply:
Done